### DISCUSSION TITLE
Improve changelogs

### DISCUSSION THREAD

**Discussion by snaptopixel** [inertiajs/inertia#277] [Ideas]
It's a bit hard to glean what's been changed from version to version. The various packages should have auto-generated changelog.md files.

**Comment by snaptopixel** [comment]
I see the notes now in the releases view which is nice, changelogs would be great too :)

**Comment by claudiodekker** [comment]
Hi @snaptopixel, this is actually something that we have been thinking about a bit recently. While the Github release notes are good enough for now, I'm going to leave this one open as a reminder of this, as well as to provide one centralized place to collect thoughts and ideas on this 👍

**Comment by snaptopixel** [comment]
Sounds great @claudiodekker, FWIW, I'm using semantic-release with https://github.com/marketplace/actions/action-for-semantic-release and it's p nice. Not in a monorepo though.

**Comment by d13r** [comment]
Would it be possible to make it more obvious that the changelog is on the Releases page? It's a bit hidden away in the GitHub UI, so it never occurred to me to look there. I looked in the repo for a CHANGELOG.md first, then the README file, then on the website, then in the Issues because I was going to raise the same issue. Thanks!
